_sass/minimal-mistakes/_masthead.scss

```diff
@@ -53,6 +53,10 @@
   z-index: 20;
 }
 
+.site-subtitle {
+  font-size: $type-size-9;
```

Comment:
This font-size is way too small, you can't read it. It'll also ding you on accessibility. I'd stick with `$type-size-8` as the smallest size you'd want to go.

Reply:
Hm, ok. On my own site it actually looks ok, as I really just want it as a subtext.
But sure.